### DISCUSSION TITLE
All Fan headers stop as soon it's enabled (switch on) except for the GPU.

### DISCUSSION THREAD

**Discussion by JuJu30420** [Rem0o/FanControl.Releases#893] [Q&A] [answered]
I have an with all my fans, except for the GPU. As soon as I enable the fan (switch ''on'' for CPU fan, or any other fan header) the fan just stops. It starts again when I switch off the fan header. It starts happening since couple days. Some say it's a Windows update and to fix it is to re-enabling .NET Framework 3.5, but it's already enabled.. I tried to reinstall it manually, but same issue. If this can't work, will need to use another software. I tried to set the fans header on Auto, PWM, standard profile and manual fan, nothing work. When the switch is disable, it show the fan speed and when switch on it goes to 0 RPM.

Any advise on this?

**Comment by JuJu30420** [comment, marked as answer]
I fixed my issue and it's possible it will work for others. I tried everything possible.

- I stopped the AsusFan Service, but no effect.
- I tried all settings in bios for fans but no effect.
- I tried older version but no effect.
- I tried to reinstalled .net but no effect.

I was about to install Asus Suite to control the fans, but I already have Asus Armory Crate witch has basic fan control. 
I also was about to uninstall MSI Afterburner (the thing is, it was installed and was working well with Fan Control so I was sure it was not this but we never know).

**solution**
Then I got an idea, I made sure the AsusFan Service was running, then I chose any fan setting (I chosed Standard) in Armory Crate (under Dashboard tab). The fan immediately start turning. I was able to control the fans in Fan Control and after some test all profile are working and respecting the Curves. I rebooted to make sure, and Fan Control initialise normally and all fans fallow the fan curve and are running with the switch ON.
** **

Issue FIXED by this solution.

**Reply by Rem0o** [reply]
So some Asus boards need the service to run for fans to be controllable at all?
